cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto

```diff
@@ -1,5 +1,5 @@
 name: "Choosing web fonts: A beginner's all-in-one guide"
-authors: "Dave Crossland"
+authors: "Charles Berret"
 reviewers: "Dave Crossland"
```

Comment:
Yuin Chien, Joana Correia, Dave Crossland, Natanael Gama, Octavio Pardo, Eben Sorkin, and Eduardo Tunni

Reply:
@davelab6 do we know about personal sites for these reviewers? I can add them in as well

Reply:
Reviewer personal sites can be added in a followup after confirming them with Dave async